index.html

```diff
@@ -161,6 +161,41 @@ <h1>The Registration Process</h1>
 the modification request adheres to the following rules:
     </p>
 
+    <p>
+      The JSON representation of a DID Document is the same document (byte for byte equivalent) 
```

Comment:
Except for the `@context` field, which is not found in JSON-only representations.

Reply:
I prefer to drop the distinction between JSON and JSON-LD and not define a data model that is exactly the same as another with a missing property....

Reply:
to be clear, I am proposing removing the "JSON Only" representation... by clarifying the registration process does not require them to do anything special, just use the JSON-LD representation... which people are actually contributing to.

Reply:
wasn't this the whole impetus for moving to an abstract data model? not having to add in a `@context` field to claim interoperability?

we removed the context property because we explicitly _do not_ use LD and don't want to add confusing.

Reply:
Resolving, as this language has been refined further.